src/app/base/components/AppSideNavigation/AppSideNavigation.tsx

```diff
@@ -80,6 +81,9 @@ const AppSideNavigation = (): JSX.Element => {
     dispatch(controllerActions.fetch());
   }, [dispatch]);
 
+  const virshKvms = useSelector(podSelectors.virsh);
+  const hasVirsh = virshKvms.length > 0;
```

Comment:
In order to be able to check thi (and display the virsh nav item when available), the pods need to be loaded. We need to make sure we fetch them regardless of which page we're on (atm it only happens on some pages).

Also it might be a good idea to use optimistic updates (meaning display first, hide later once data is loaded).

Reply:
Ahhh righto, would that just be in a useEffect? Something like:

```ts
useEffect(() => {
    dispatch(podActions.fetch());
  }, [dispatch]);
```